src/script/interpreter.cpp

```diff
@@ -14,6 +14,8 @@
 #include "script/script.h"
 #include "uint256.h"
 
+#include "sodium.h"
+
```

Comment:
This got added to the file accidentally or something. I don't think it's needed.